package.json

```diff
@@ -42,6 +42,7 @@
     "eslint-plugin-babel": "^4.0.1",
     "eslint-plugin-google-camelcase": "0.0.2",
     "extract-text-webpack-plugin": "^2.0.0-rc.3",
+    "first-mate": "7.3.0",
```

Comment:
Why is this dependency needed?

Reply:
That's explained in the [commit message](https://github.com/4mfie/ampstart/commit/703419abf380000f431bf023c709d3254b989c03). I'll paste its contents below for the sake of convenience.

The `highlights` package depends on a version range of the `first-mate` package, which currently resolves to v7.4.0. This version of `first-mate` depends on `oniguruma` v7.2.0. This version of `oniguruma` is broken. I am locking `first-mate` at v7.3.0, which uses the previous functioning version of `oniguruma` (v7.1.0) in order for `npm i` to complete.